tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -342,9 +342,10 @@ class WorksheetSuite extends BaseSuite {
         |def fn(using c: C) = ()
         |val xx = fn
         |""".stripMargin,
-    """|dotty-ambiguous-implicit:8:10: error: ambiguous implicit arguments: both object c1 in class App and object c2 in class App match type App.this.C of parameter c of method fn in class App
+    """|dotty-ambiguous-implicit:8:12: error:
+       |ambiguous implicit arguments: both object c1 in class App and object c2 in class App match type App.this.C of parameter c of method fn in class App
        |val xx = fn
-       |         ^^
+       |           ^
```

Comment:
I thought I broke something, but it seems to be this on the latest:

```
❯ cs launch scala3-repl
scala> abstract class C:
     |   val x: Int
     | given c1 : C with
     |   val x = 1
     | given c2 : C with
     |   val x = 2
     | def fn(using c: C) = ()
     | val xx = fn
8 |val xx = fn
  |           ^
  |ambiguous implicit arguments: both object c1 and object c2 match type C of parameter c of method fn
```

Reply:
We worked around the issue, because this was causing issues for edit distance. If it's on a non-existing position we cannot match it to any real token. https://github.com/scalameta/mdoc/pull/386

Reply:
Could you open up an issue to figure it out later on?